assets/theia.svg

```diff
@@ -0,0 +1,37 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000" height="1000">
+<!--
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
```

Comment:
Should put for the Copyright 2020 and not 2019 anymore for new file

Reply:
I think we keep the date that the file was written (which in this case was 2020).

Reply:
> Should put for the Copyright 2020 and not 2019 anymore for new file

I believe we keep the date the file was written (in this case it was 2019).

Reply:
It looks like this file was a new file, that why I said the Copyright should be 2020. If it is just a copy of an old file, then ok